<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditUmlClassInfoDialog</class>
 <widget class="QDockWidget" name="EditUmlClassInfoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>230</width>
    <height>298</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>230</width>
    <height>290</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>UML Class Infomation</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <widget class="QTextEdit" name="classNameArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::NoWrap</enum>
    </property>
    <property name="placeholderText">
     <string>Class Name</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="classAttributesArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>211</width>
      <height>101</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::NoWrap</enum>
    </property>
    <property name="placeholderText">
     <string>Attributes</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="classMethodsArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>211</width>
      <height>101</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::NoWrap</enum>
    </property>
    <property name="placeholderText">
     <string>Methods</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>classNameArea</tabstop>
  <tabstop>classAttributesArea</tabstop>
  <tabstop>classMethodsArea</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
